src/containers/Blog/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { Row, Col, Container, Image } from "react-bootstrap";
+import "./index.css";
+import LoadingPage from "../../components/LoadingPage";
+
+const Blog = () => {
+  const [blog, setBlog] = React.useState();
+  let params = useParams();
+  const storyID = params.id;
+  console.log(params.id);
+  const _storyURL = `https://public-api.wordpress.com/wp/v2/sites/nobarriers650349093.wordpress.com/posts/${storyID}`;
```

Comment:
lets move this const inside the useEffect since we only use it there and we want its scope as small as possible

Reply:
moved

Reply:
Moved